src/utils/ddc_aliases.hpp

```diff
@@ -99,3 +100,328 @@ class Vector : public ddc::detail::TaggedVector<ElementType, Dims...>
 /// A type describing a vector whose elements are doubles e.g. (E_x, E_y)
 template <class... Dims>
 using DVector = Vector<double, Dims...>;
+
+
+//----------------------------------------------
+//  Aliases for batched 1D spline builder
+//----------------------------------------------
+/**
+ * @brief A class for creating a 1D spline builder.
+ * @see SplineBuilder
+ */
+template <
+        class ExecSpace,
+        class BSplines,
+        class InterpolationGrid,
+        ddc::BoundCond BCLBound,
+        ddc::BoundCond BCUBound,
+        ddc::SplineSolver SolverType,
+        class IdxRangeType>
+class GetSplineBatchedBuilder1D;
```

Comment:
I think these classes should be confined to a namespace (e.g. `namespace details{`) as they only exist as a tool to allow us to define the type aliases (e.g. `get_spline_batched_builder1d_t`).